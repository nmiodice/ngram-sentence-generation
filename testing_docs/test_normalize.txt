12.34.?"{}+_:>?"{~}      \\ }{}
12.34.?"{}+_:>?"{~}      \\ }{}
Hello world. Hello World. HELLO WORLD. hello, world. 





world hold on




to me

234234 {}
++__++


please FFF F 8383


This should be the only example of a normal sentence here