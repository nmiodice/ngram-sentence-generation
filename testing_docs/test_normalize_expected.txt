

hello world hello world hello world hello world





world hold on




to me





please fff f


this should be the only example of a normal sentence here